### ISSUE TITLE
API, not loading at all, screen flashes, and API copy are my errors

### ISSUE THREAD

**Issue by CurrentWave** [FreeTubeApp/FreeTube#1718]
Hello, 
Just installed latest FreeTube app on Linux Mint 20.2 Cinnamon DE a few days ago (Sep 2021)

And started watching and adding subscriptions, all was well and videos played just fine.
Kept adding subscriptions - is there a limit?  I have about 30 now.  Small issues starting showing up and getting progressively worse.....

**Behavior of the program**
All of sudden, sometimes when loading a video, it would get stuck loading and loading - so I would do a shift+ctrl+r to refresh the screen, and the video would load just fine.  This now was a consistent glitch, but I ignored it.

Next, was when I first clicked on a video in my subscriptions I would get those popup bubbles saying something about the API not loading/working/something, falling back to local api  & click to copy, error code 429 and socket dls disconnected before.... I cannot catch these errors showing for seconds.

I tried changing the Current Invidious Instance, but did not set it as I do not understand any of this.  This did not help.  I also tried reading about FT, because I was not understanding why things were going wrong and what I should do, I was looking for how things work and how to troubleshoot.  I found nothing helpful to my situation. 

And now the install is super glitchy - I never know what will load, what will play and what will not.  Black bubbles flash, pop in and out, the screen flashes like it is going to go to a new screen but then flashes back and does nothing. 

**Expected behavior**
That videos will play when I click on them.
That if my instance goes bonky the software will select a new stable one for me.  I am not a programmer and do not understand most of the choices in Settings.

**To Reproduce**
You cannot - obviously my app is degrading (falling apart as I use it).  All I did was install, search for videos, subscribe to channels and watch videos.  I didn't go into settings until it was already glitching, where I select to hide videos I have already watched and to use a different instance from the choices offered.
Is there a way to save one's user, and reinstall FT new, without loosing all the subscriptions?

**Link to video**
I see no patterns to my glitches except for one - when I first load the program all appears well, but as I select and watch videos things get worse and worse....

**Screenshots**
I tried, but noting showed up in my screenshot.  I will work on trying to grab some shots as the errors flash by pretty fast.  Is there an error log file?

**Environment Information (please complete the following information):**
 - OS: Linux Mint 20.2 Cinnamon
 - FreeTube version: v0.14.0 Beta
 - Installation = .deb
 - Primary API Used: invidious.exonip.de

**Comment by jasonhenriquez** [comment]
About 429 errors: https://docs.freetubeapp.io/usage/common-issues/. To see what errors are being logged, if you could open the dev console (Ctrl + Shift + i) and share any errors that appear, it would be greatly appreciated. About exporting subscriptions: Go to the settings page under Data Settings and select "Export Subscriptions" to a format of your choice. You can import those subscriptions, in case you ever lose them, from the "Import Subscriptions" button in the same Data Settings section.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi, @CurrentWave i just wanted to note that its not best practice to put multiple issues into one bug report. Its ok for now but please do not do this in the future.

**Comment by CurrentWave** [comment]
Hum, multiple issues... it all looks like one to me :) but I believe I've accidentally narrowed the problems down, read on.

**So I went - and picked a new Instance -->** invidious.kavin.rocks and made it my default, just to try doing something and see what the outcome would be.

All those errors in bubbles stopped - Yea!  However, the screen flashing when loading a video remained sometimes (One error?), but this is not big issue preventing watching so I am very happy with this improvement.

@jasonhenriquez - Thanks for your reply, I did use the link and do my reading this morning.
Then I did as you suggested, and used the dev console : here is a representation of two errors that just keep repeating for different videos from this morning------------------

invidious.kavin.rocks/api/manifest/dash/id/rZTulxVuwbQ.mpd:1 Failed to load resource: the server responded with a status of 403 ()
/app/freetube/resources/app.asar/node_modules/video.js/dist/video.cjs.js:156 VIDEOJS: ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The media could not be loaded, either 
because the server or network failed or because the format is not supported. 
MediaError
(anonymous) @ /app/freetube/resources/app.asar/node_modules/video.js/dist/video.cjs.js:156
renderer.js:2 MediaError
renderer.js:2 Unable to play dash formats.  Reverting to legacy formats...
index.html#/watch/rZTulxVuwbQ:1 Uncaught (in promise) DOMException: Failed to load because no supported source was found.
renderer.js:2 Uncaught TypeError: Cannot read properties of null (reading 'qualityLevels')
    at renderer.js:2
renderer.js:2 Uncaught TypeError: Cannot read properties of null (reading 'videoWidth')
    at o.checkAspectRatio (renderer.js:2)
    at renderer.js:2

invidious.kavin.rocks/api/v1/channels/playlists/UC-eljawzQ2r2Cnm9g7X3VaA?sort_by=last:1 Failed to load resource: the server responded with a status of 502 ()
renderer.js:2 Object
renderer.js:2 error
renderer.js:2 https://invidious.kavin.rocks/api/v1/channels/playlists/UC-eljawzQ2r2Cnm9g7X3VaA?sort_by=last
renderer.js:2 
renderer.js:2 Object
renderer.js:2 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'error')
    at renderer.js:2

Thanks for your time ~
